front/src/components/app/index.js

```diff
@@ -0,0 +1,3 @@
+import App from './App';
```

Comment:
Why do we need `App.js` at all? Please, move all the code from `App.js` to the `index.js` inside app folder

Reply:
done